src/containers/Auth/LoginScreen.js

```diff
@@ -102,6 +102,10 @@ export const LoginScreen = () => {
               field={t('Username or E-mail address:')}
               name='username'
               value={values.username}
+              accessibilityHint={t('type in your password')}
```

Comment:
eeeesh this is tricky as I agree it is now a long list so I get wanting to tidy but personally I prefer not to abstract away as then you are looking in other places to figure out what belongs where and editing the form fields in 2 places instead of 1 so i'm not sure